dask/dataframe/core.py

```diff
@@ -357,6 +357,39 @@ def _rebuild(self, dsk, *, rename=None):
     def _constructor(self):
         return new_dd_object
 
+    @property
+    def divisions(self):
+        """
+        Tuple of ``npartitions + 1`` values, in ascending order, marking the
+        lower/upper bounds of each partition's index. Divisions allow Dask
+        to know which partition will contain a given value, significantly
+        speeding up operations like `loc`, `merge`, and `groupby` by not
+        having to search the full dataset.
+
+        Example: for ``divisions = (0, 10, 50, 100)``, there are three partitions,
+        where the index in each partition contains values [0, 10), [10, 50),
+        and [50, 100), respectively. Dask therefore knows ``df.loc[45]``
```

Comment:
```suggestion
        and [50, 100], respectively. Dask therefore knows ``df.loc[45]``
```
@douglasdavis good catch, I think this is what you're suggesting?

Reply:
Yes!